frontend/src/e2e-test/pages/employee/documents/child-document.ts

```diff
@@ -12,6 +12,14 @@ export class ChildDocumentPage {
   readonly previewButton = this.page.findByDataQa('preview-button')
   readonly returnButton = this.page.findByDataQa('return-button')
 
+  async assertDocumentVisible() {
```

Comment:
Suggestion: since the status field is already public, just call `waitUntilVisible` / `waitUntilHidden` directly in tests. These "one-liner" wrappers are 99% of the time unnecessary boilerplate.

In this case there's a tiny bit of value in the terminology change: it's different to wait for "status to be visible" vs "document to be visible". But I think this value is very small

Reply:
Fixed